connection.go

```diff
@@ -31,6 +31,7 @@ type mysqlConn struct {
 	sequence         uint8
 	parseTime        bool
 	strict           bool
+	warningCount     uint16
```

Comment:
The fields are sorted by size for efficient struct packing. The `warningCount` field should moved up (before `flags` I think)